lib/rust/parser/src/syntax/tree/block.rs

```diff
@@ -141,7 +141,6 @@ where I: Iterator<Item = Line<'s>>
 #[derive(Debug)]
 enum Prefix<'s> {
     BuiltinAnnotation { node: Box<AnnotatedBuiltin<'s>>, span: Span<'s> },
```

Comment:
Just a thought, but would it be potentially desirable to do similar refactoring for annotations? That would completely get rid of the `Prefix` type.

Reply:
That's the plan, but wasn't needed for the documentation editor (I had to draw the line somewhere). It will be done as part of #5453.